wfe/wfe_test.go

```diff
@@ -1932,3 +1956,101 @@ func TestDeactivateRegistration(t *testing.T) {
 		  "status": 403
 		}`)
 }
+
+func TestKeyRollover(t *testing.T) {
```

Comment:
This tests several cases in one long function. Could you do one of these two things?

 - Split it into one function per case, like TestKeyRolloverParseError, TestKeyRolloverNewJWKBad, with setup code factored out into a helper as necessary
 - Turn it into a table-based test.